cove_bods/templates/cove_bods/additional_checks_table.html

```diff
@@ -304,18 +304,18 @@
         {% elif additional_check.type == 'unknown_schema_version_used' %}
             <tr>
                 <td>
-                    {% blocktrans %}This data attempted to use a schema version that was not recognised.{%endblocktrans%}
+                    {% blocktrans %}<code>bodsVersion</code> not valid. Check that the value is correctly formatted.{% endblocktrans %}
                 </td>
                 <td>
-                    {{ additional_check.schema_version }}
+                    <code>bodsVersion</code>: {{ additional_check.schema_version }}
                 </td>
                 <td>
                 </td>
             </tr>
         {% elif additional_check.type == 'statement_is_beneficialOwnershipOrControl_but_no_person_specified' %}
             <tr>
                 <td>
-                    {% blocktrans %}This Ownership-or-control Statement contains interests where beneficialOwnershipOrControl is true. Therefore interestedParty must reference a Person Statement.{%endblocktrans%}
+                    {% blocktrans %}<code>Ownership-or-control statement</code> asserts <code>beneficialOwnershipOrControl</code> is 'true' but <code>interestedParty</code> does not reference a Person statement. Check that information is correctly represented.{% endblocktrans %}
```

Comment:
```suggestion
                    {% blocktrans %}<code>Relationship statement</code> asserts <code>beneficialOwnershipOrControl</code> is 'true' but <code>interestedParty</code> does not reference a Person statement. Check that information is correctly represented.{% endblocktrans %}
```

Reply:
This check only works for pre BODS 0.4 hence the terminology.